src/main/app/document/DocumentManagementClient.ts

```diff
@@ -31,7 +32,13 @@ export class DocumentManagementClient {
     }
 
     const response: AxiosResponse<DocumentManagementResponse> = await this.client.post('/documents', formData, {
-      headers: { ...formData.getHeaders(), 'user-id': this.user.id },
+      headers: {
+        ...formData.getHeaders(),
+        'user-id': this.user.id,
+      },
+      maxContentLength: Infinity,
+      maxBodyLength: Infinity,
```

Comment:
maxBodyLength defines the max size of the http request content. Set to infinity because the default is 10Mb and we are already limiting the file upload size in upload-manager/index L28.